### ISSUE TITLE
Make generic error handler

### ISSUE THREAD

**Issue by skurnevich** [zowe/zen#4]
Need to create new components to handle and display errors in a unified way, like toast notification

**Comment by DivergentEuropeans** [comment]
@skurnevich  I'm going to expand the scope a bit. Not only will it include errors, but it will probably be good to include warnings and success statuses like this: 

![image](https://github.com/zowe/zen/assets/20528015/0aa6daf7-5bdf-47fc-830a-ad45445c317a)

Thoughts? @skurnevich

**Comment by skurnevich** [comment]
@DivergentEuropeans yes i would agree, we should use same notification for any kind of information. 

Point of discussion there is how should it look, and where it should be placed 

Most obvious place is bottom-right corner
![Screenshot 2023-09-04 at 16 57 01](https://github.com/zowe/zen/assets/61471197/970365fb-8796-4b3d-9bd5-2b35dedfdf3e)
![Screenshot 2023-09-04 at 16 57 18](https://github.com/zowe/zen/assets/61471197/72b782ed-ceed-4fda-85e2-797b4c594952)

But it also would be useful to have some configuration help on some stages. And it also a good candidate to take bottom left corner 
![Screenshot 2023-09-04 at 17 04 37](https://github.com/zowe/zen/assets/61471197/d45d4d23-bcb8-4f2a-9643-8db20b64aa4e)

So we'll need to find some other way to display help and tips
Or move notification to the top under the header
![Screenshot 2023-09-04 at 17 01 25](https://github.com/zowe/zen/assets/61471197/891a62de-f7e3-447d-b898-4f6d6ae81d1d)
